astropy_timeseries/binned.py

```diff
@@ -160,3 +161,92 @@ def __getitem__(self, item):
                                                  keys=self.groups._keys)
                 return out
         return super().__getitem__(item)
+
+    @classmethod
+    def read(self, filename, time_bin_start_column, time_bin_end=None,
```

Comment:
As for the sampled case, ``time_bin_start_column`` should be optional as there might be file formats for which we define readers which only need the filename. Also ``time_bin_end`` should be ``time_bin_end_column``.

Reply:
Woops, I renamed it and changed the messages.